jerry-core/ecma/base/ecma-globals.h

```diff
@@ -741,18 +741,12 @@ typedef uintptr_t ecma_external_pointer_t;
   */
 typedef struct
 {
-  uint16_t status_flags;            /**< various status flags */
+  uint16_t size;                    /**< real size >> MEM_ALIGNMENT_LOG */
+  uint16_t refs;                    /**< reference counter for the byte code */
+  uint16_t status_flags;            /**< various status flags. */
```

Comment:
The doc of this field should be amended with some short description of the valid flags (e.g., "various status flags (CBC_CODE_FLAGS_xxx)")